app/javascript/App/bundles/NewsfeedPage/NewsBody.tsx

```diff
@@ -165,6 +166,9 @@ export default class NewsBody extends React.Component<Props, State> {
             }}
           </RailsConsumer>
         </div>
+
+        <br />
+        <CallToAction />
```

Comment:
should this only be here if not logged in?

Reply:
Was hoping to not have to prop drill `loggedIn` but it's a better UX so will just bite the bullet, thanks.